src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int sizeOfList;
```

Comment:
if it's a field of list, it's obvious that it's the size of the list :) let's keep it simple
```suggestion
    private int size;
```